tests/python-pytest/onnx/test_operators.py

```diff
@@ -1858,3 +1858,19 @@ def rand_check(out):
     def rand_check_nd(out):
         return rand_check(out.asnumpy())
     op_export_test('sample_multinomial', M, [x], tmp_path, mx_map=rand_check_nd, onnx_map=rand_check)
+
+
+@pytest.mark.parametrize("dtype", ['float32', 'int32', 'int64'])
```

Comment:
Is the customer use case covered by the unit tests?

Reply:
Customer has not shared their use case yet